docs/fledge_architecture.rst

```diff
@@ -95,16 +96,31 @@ A sample of existing Filters:
 
 **Rate**: buffer data but don’t send it, then if an error condition occurs, send the previous data.
 
-Event Engine
-============
+**Contrast**: Enhance the contrast of image type data
```

Comment:
This filter plugin is not available at Fledge side.

Reply:
Not too worried about this